Lib/pdb.py

```diff
@@ -237,6 +240,44 @@ def namespace(self):
         )
 
 
+class _ZipTarget(_ExecutableTarget):
+    def __init__(self, target):
+        import runpy
+
+        self._target = os.path.realpath(target)
+        sys.path.insert(0, self._target)
```

Comment:
Should this be removed from the path later?

Reply:
The current Python basically does this for zipapp and the path stays until the end of the program. For script targets, we modified `sys.path` and never changed it back and I think we do not need to revert the path here after the program either. (during program this is required)